traitsui/tests/test_editor.py

```diff
@@ -0,0 +1,733 @@
+#  Copyright (c) 2008-19, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in enthought/LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+#  Author: Corran Webster
+#  Date:   August 12, 2019
+
+import unittest
+
+from traits.api import (
+    Any, Bool, Event, Float, HasTraits, Int, List, Undefined
+)
+from traits.trait_base import xgetattr
+from traits.testing.unittest_tools import UnittestTools
+
+from traitsui.context_value import ContextValue, CVFloat, CVInt
+from traitsui.editor import Editor
+from traitsui.editor_factory import EditorFactory
+from traitsui.handler import default_handler
+from traitsui.ui import UI
+
+from ._tools import is_current_backend_null
```

Comment:
Can `_tools` be promoted to public to avoid the private import? (Not in this PR.)

Reply:
Yes, I'd like that - in fact it probably belongs in a traitsui.testing module or some such.